### PR TITLE
BETA: Register matrixcoder.is-a.dev

### PR DESCRIPTION
**Pull request by MatrixCoder0101**
Added `matrixcoder.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/matrixcoder.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "MatrixCoder0101",
+        "email": "tkgoutamm625@gmail.com"
+    },
+    "record": {
+        "CNAME": "matrixcoder0101.github.io"
```
Comment (DEV-DIBSTER) [inline domains/matrixcoder.json:7]: No content is displayed, just default Vercel page.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/matrixcoder.json (hunk 1 above)
Comment (MatrixCoder0101) [reply domains/matrixcoder.json:7]: Now I am working on my portfolio

Comment (DEV-DIBSTER) [reply domains/matrixcoder.json:7]: It's just a blank NextJS page.

Comment (MatrixCoder0101) [reply domains/matrixcoder.json:7]: > It's just a blank NextJS page.

Sir Please Recheck and Approve my Request.

Comment (MatrixCoder0101) [reply domains/matrixcoder.json:7]: Sir Please Recheck now

Comment (DEV-DIBSTER) [reply domains/matrixcoder.json:7]: Looks good.

**Review by DEV-DIBSTER — APPROVED** [review]

File: domains/matrixcoder.json (hunk 1 above)
Comment (MatrixCoder0101) [reply domains/matrixcoder.json:7]: **Can you help me Sir how I apply this domain in GitHub pages 🤔**

**Pull request merged**